system/coverage/browser/CodeBrowser.cfc

```diff
@@ -98,11 +90,13 @@ component accessors=true {
 				var lineNumbersBGColorsJSON = SerializeJSON( lineNumbersBGColors );
 				var fileContents            = fileRead( fileData.filePath );
 
-				savecontent variable="local.fileTemplate" {
+				var brush = right( fileData.relativeFilePath, 4 ) == ".cfm" ? "coldfusion" : "javascript";
+
+				savecontent variable="fileTemplate" {
```

Comment:
please change this back to `local`, if not it will be placed as `variables`

Reply:
I am all for consistency and good looking code, but we all have different habits and doing this kind of thing manually is very tedious. I will do my best to break my habits and stick to your standards as I think they are better than what I am used to. With that said, do you have anything (editor rules, Prettier, CFLint, etc.) currently set up in your environment to automate this kind of thing?

Reply:
This looks great. Did you format it manually, or is it automated?
<img width="528" alt="screen shot 2019-01-29 at 12 12 23 pm" src="https://user-images.githubusercontent.com/6798554/51937357-48494d80-23bf-11e9-83d3-18a53ccef572.png">

Reply:
Just saw the beautify for VS Code section here https://github.com/Ortus-Solutions/coding-standards.